c2corg_ui/static/js/documentediting.js

```diff
@@ -565,6 +573,43 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
 
 
 /**
+ * @export
+ */
+app.DocumentEditingController.prototype.openDateStart = function() {
+  this.dateStartOpened = true;
+};
+
+
+/**
+ * @export
+ */
+app.DocumentEditingController.prototype.openDateEnd = function() {
+  this.dateEndOpened = true;
+};
+
+
+/**
+ * @export
+ * @suppress {missingProperties}
+ */
+app.DocumentEditingController.prototype.formatDate = function(date) {
```

Comment:
> I'm a bit reluctant with having document_type specific functions and properties in this controller, that should be kept as generic as possible. Should we consider using inheriting controllers when the generic editing controller is not enough?

I have created an issue for this: https://github.com/c2corg/v6_ui/issues/246

Perhaps you could make the suggested change for ``formatDate`` anyway?